apps/web/src/pages/blob/[hash].tsx

```diff
@@ -87,9 +88,33 @@ const Blob: NextPage = function () {
       });
     }
     detailsFields.push(
-      { name: "Versioned Hash", value: blob.versionedHash },
-      { name: "Commitment", value: blob.commitment },
-      { name: "Proof", value: blob.proof }
+      {
+        name: "Versioned Hash",
+        value: (
+          <div className="flex items-center gap-2">
+            {blob.versionedHash}
+            <CopyToClipboard value={blob.versionedHash} />
+          </div>
+        ),
+      },
+      {
+        name: "Commitment",
+        value: (
+          <div className="flex items-center gap-2">
+            {blob.commitment}
+            <CopyToClipboard value={blob.commitment} label="Copy commitment" />
+          </div>
+        ),
+      },
+      {
+        name: "Proof",
+        value: (
+          <div className="flex items-center gap-2">
+            {blob.proof}
+            <CopyToClipboard value={blob.proof} label="Copy proof" />
+          </div>
```

Comment:
All of them share the same structure and styles. I would create a common stateless component

Reply:
Done [386cd1a](https://github.com/Blobscan/blobscan/pull/478/commits/386cd1aa22f7d42021daf8de6bb6a77880df3813)